index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./styles/style.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+			integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA=="
+			crossorigin="anonymous" referrerpolicy="no-referrer" />
+	<title>My Portfolio</title>
+</head>
+
+<body>
+	<header>
+		<nav>
+			<ul>
+				<li class="nav-welcome"><a href="#">welcome</a></li>
+				<li><a href="#" class="nav__button"><i class="fas fa-bars"></i></a></li>
+			</ul>
+		</nav>
+	</header>
```

Comment:
- [ ] Great job structuring the header! It fulfills the project requirements. However, an unambiguous and recommended way to structure it is as below .
```
<header>
  <a href="#"> -- logo goes here -- </a>
  <nav>
    <button> -- menu icon goes here -- </button>
  </nav>
</header>
```